prod-flatcar/wasmtime.yaml

```diff
@@ -55,4 +54,25 @@ systemd:
         OnCalendar=*-*-* *:*:00
         Persistent=true
         [Install]
-        WantedBy=timers.target
\ No newline at end of file
+        WantedBy=timers.target
+
+    - name: sysext-update-watcher.path                                                   
+      enabled: true
+      contents: |                                                                        
+        [Unit]
+        Description=Trigger a restart of systemd-sysext to auto-merge new / updated extensions.
+        [Path]
+        PathModified=/etc/extensions
+        Unit=sysext-restart.service
+        [Install]
+        WantedBy=paths.target
+
+    - name: sysext-restart.service
+      enabled: false
+      contents: |
+        [Unit]
```

Comment:
This can race with the update still being in progress. Adding `After=sysext-wasmtime-update-check.service` should help.

Reply:
And maybe even `Conflicts=sysext-wasmtime-update-check.service` which should ensure that only one of them can run at the same time.